src/lib/url.js

```diff
@@ -100,17 +100,19 @@ export const compileQueryParameterFamily = (baseName, query) => {
   return compile(baseName, query);
 };
 
-export const compileQuery = (baseName, query) => {
+export const compileQueryParameter = (baseName, query) => {
+  const queryValue = query[baseName];
```

Comment:
Perfect, this is what I was expecting. I know it's a little weird, but from the outside this method signature now parallels `parseQueryParameterFamily`